src/components/Experiment/QuickInput/Tabs/SampleInput/SampleInputsTab.js

```diff
@@ -107,10 +111,26 @@ export default function SampleInputsTab(props) {
         return (
             <button onClick={() => onSampleInputClickPreview(index, url)} key={index} className={getElement(getInputClassName(url))}>
                 <video src={url.src} alt={url.alt} autoPlay muted={true} loop className={getElement("sample-video-content")} />
+                <URLInputPreview task={task} index={index} inputPreviewProps={props.inputPreviewProps} inputType={TaskInputTypes.Video} selectedInputs={props.values} />
```

Comment:
@ShivanshShalabh Can you confirm that this change is OK? I moved the location of the URLInputPreview because it was causing errors with VideoClassification